flink-staging/flink-gelly/src/test/java/org/apache/flink/graph/test/operations/GraphOperationsITCase.java

```diff
@@ -266,6 +266,47 @@ public void testUnion() throws Exception {
 					"6,1,61\n";
 	}
 
+    @Test
+    public void testDifference() throws Exception {
+		/*
+		 * Test difference()
+		 */
+        final ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+
+        Graph<Long, Long, Long> graph = Graph.fromDataSet(TestGraphUtils.getLongLongVertexData(env),
+                TestGraphUtils.getLongLongEdgeData(env), env);
+
+        List<Vertex<Long, Long>> vertices = new ArrayList<Vertex<Long, Long>>();
+        List<Edge<Long, Long>> edges = new ArrayList<Edge<Long, Long>>();
```

Comment:
I would put these in TestGraphUtils, one remove is fine, but three can make the code  a bit difficult to read :)